securitycenter/CHANGELOG.md

```diff
@@ -4,6 +4,33 @@
 
 [1]: https://pypi.org/project/google-cloud-securitycenter/#history
 
+## 0.2.0
+
+03-12-2019 17:09 PDT
+
+
+### Implementation Changes
+- Remove unused message exports. ([#7274](https://github.com/googleapis/google-cloud-python/pull/7274))
+- Trivial gapic-generator change. ([#7233](https://github.com/googleapis/google-cloud-python/pull/7233))
+- Protoc-generated serialization update, docstring tweak. ([#7094](https://github.com/googleapis/google-cloud-python/pull/7094))
+
+### New Features
+- Security Center: Generate v1 ([#7495](https://github.com/googleapis/google-cloud-python/pull/7495))
```

Comment:
Drop the "Security Center:" prefix.  I think this would be better as:
```suggestion
- Add support for `v1` API. ([#7495](https://github.com/googleapis/google-cloud-python/pull/7495))
```